CONTRIBUTING.md

```diff
@@ -20,14 +20,7 @@ pnpm run compile
 ## Start with example
 
 ```shell
-pnpm run examples:dev:01_template
-```
-
-or if you want to run a specific example:
-
-```shell
-cd examples/01_template
-pnpm run dev
+pnpm -F 01_template dev # to run examples/01_template in dev mode
```

Comment:
This is so good!
Can you add an instruction for the `pnpm run examples:prd:01_template` equivalent?

Reply:
is this good?

```shell
pnpm -F 01_template dev # to run examples/01_template in dev mode
```

`build` and `start` can be run with the same pattern:

```shell
pnpm -F 01_template build
pnpm -F 01_template start
```

---

Or do you want to explicitly call out:

`pnpm -F 01_template build &&  pnpm -F 01_template start`

Reply:
Hm, whichever is fine. I'd probably use some hack personally. So, just consider a better way for docs.

Reply:
updated to the above without the `&&` 

happy hacking haha